### ISSUE TITLE
from_Solis: smarter import of array indices

### ISSUE THREAD

**Issue by ddkohler** [wright-group/WrightTools#972]
Retain true ccd indices on cropped files by using the metadata.  Currently handles cropped data correctly only for `xindex`.

Rotated files is trickier to handle; perhaps apply rotations by applying different transform (e.g. default axes are `[xindex, yindex]`, but if the metadata indicates clockwise rotation, use axes `[yindex, -xindex]`.  Unsure as of yet whether the raw output is already rotated.

**Comment by ksunden** [comment]
Is this information even available in the file? I see in he table at the bottom some things (e.g. rotations/horizontal/vertical flips...) Perhaps more is added in other readout modes, but in full resolution image mode (which is the only mode we test against/have distributed files for), there is no indication of position.

**Comment by ddkohler** [comment]
yeah, the field `{left,right,bottom,top}` is there (at least when cropped).  

I'll include a new dataset in PR.
